capirca/lib/nacaddr.py

```diff
@@ -323,8 +323,12 @@ def _CollapseAddrListInternal(addresses, complements_by_network):
         prev_addr.AddComment(addr.text)
       elif (prev_addr.version == addr.version and
             prev_addr.prefixlen == addr.prefixlen and
-            prev_addr.broadcast_address + 1 == addr.network_address and
-            prev_addr.Supernet().network_address == prev_addr.network_address):
+            # It's faster to compare integers than IP objects
+            prev_addr.broadcast_address._ip + 1 == addr.network_address._ip and  # pylint disable=protected-access
+            # Generating Super is relatively intensive compared to doing bit
```

Comment:
Nit: can you change this to "generating Supernet", I got confused and thought you meant a parent class for a second

Reply:
For sure!